webhooks-extension/docs/InstallReleaseBuild.md

```diff
@@ -0,0 +1,93 @@
+# Installing Official Releases
+<br/>
+
+Run the [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply) command to perform installation.  By default the installation will be into a namespace called "tekton-pipelines".  
+
+If the tekton dashboard has been installed into a namespace other than "tekton-pipelines", then you will need to ensure you install the webhooks-extension into the same namespace.  Instructions for installing into an alternative namespace are provided below the standard install instructions.
```

Comment:
Tekton Dashboard

Let's say Webhooks Extension everywhere and not `webhooks-extension`

Reply:
changed this occurrence to Webhooks Extension component